packages/cozy-harvest-lib/src/services/updateAccountsPassword.js

```diff
@@ -9,6 +9,10 @@ import {
   launchTriggers
 } from './utils'
 
+import logger from 'cozy-logger'
+
+logger.namespace('cozy-harvest-lib')
```

Comment:
There is a src/logger file that exports a logger is the right namepsace.

Reply:
Yes but here since we are in a service, we need to handle the stack's log format. That's why I used cozy-logger. Does minilog do it?

Reply:
Ah yes sorry, I think we could tell minilog to do it yes. At first, taking it from src/services/logger and explaining this could do the trick.

Reply:
See https://github.com/cozy/cozy-libs/commit/c368afa992f70bce622112dbd9897c2bd39c7693